tools/dev/arnref.py

```diff
@@ -0,0 +1,91 @@
+"""Build an arn-type database from aws iam reference docs
+
+Inspired by duo-labs/parliament and salesforce/policy_sentry which
+also parse the reference docs.
+
+The simplest way to run is actually checkout policy_sentry since
+it has a clean copy of the html data.
+
+```
+gh repo clone salesforce/policy_sentry
```

Comment:
What type of licence is this under?

Reply:
the policy_sentry project is mit (https://github.com/salesforce/policy_sentry/blob/master/LICENSE) , the content is just general ref docs from aws via download, those are generally apache or mit. https://github.com/awsdocs